templates/submission/status.html

```diff
@@ -2,7 +2,8 @@
 {% block js_media %}
     <script type="text/javascript" src="{{ static('event.js') }}"></script>
     {% if not submission.is_graded and last_msg %}
-        <script type="text/javascript">$(function () {
+        <script type="text/javascript">
+        $(function () {
```

Comment:
If we're fixing the spacing here, might as well fix the indentation too.

Reply:
Done